core/tests/protractor_utils/ExplorationEditorHistoryTab.js

```diff
@@ -56,6 +62,21 @@ var ExplorationEditorHistoryTab = function() {
    * Workflows
    */
 
+  this.expectCommitDatesToBeDisplayed = async function() {
+    /*
+     * This method checks if the commit dates are being displayed in
+     * List of Changes section of the history tab.
+     */
+    var numCommitDates = await datesCommitsWereSaved.count();
+    for (var i = 0; i < numCommitDates; i++) {
+      var date = await (await datesCommitsWereSaved.get(i)).getText();
```

Comment:
You only need the other `await`. `.get()` returns an ElementFinder, not a Promise